### PR TITLE
AWS org config + graph mapping questions

### PR DESCRIPTION
**Pull request by a-u-h-g**

### CODE REVIEW

**Review by aiwilliams — COMMENTED** [review]
IIRC, the integration will transfer the enablement state from the master instance config to the sub account instance configs when it creates. This has caused confusion (and frustration) for folks at times as they have had to go back an enable the integration instances. Have you observed that at all?

**Comment by aiwilliams** [comment]
Oh, just noticed the title of the PR is unrelated to the changes 😄

File: faqs/faqs-aws.md [hunk 1]
```diff
@@ -13,6 +13,27 @@ Or they can be found on the [jupiterone-aws-integration][] project on Github.
...
+During this process you will create an IAM Role for JupiterOne with specific policies attached and a 
+specific external trust ID. Please note the IAM Role name, policies, and external trust ID used. 
+Finally, make sure that in the JupiterOne application you select the option to "Auto-configure additional integrations...". 
+
+Now use your favorite infrastructure-as-code method to systematically generate an identical JupiterOne IAM Role in each of your 
+sub-accounts. Be sure to name the IAM Role identically, attach the same policies, and use the same external trust ID 
+as was used with the master account configuration. If these steps are done correctly, JupiterOne will automatically pull in all
+sub-accounts from the Organization the next time it polls your environment.
+
+[jupiterone-aws-integration]: https://github.com/jupiterone/jupiterone-aws-integration
```
[the remark this answers is not in the document]
Comment (a-u-h-g) [reply faqs/faqs-aws.md:29]: I was wondering about that, but the original had the link at the top so I just figured, maybe the want to link to something else with the same words in the future. Changed it now though

**Comment by a-u-h-g** [comment]
@aiwilliams I have not noticed the enable all sub-accounts issue, does this happen when it is set to disabled on initial config? I can make an update for that. Changed the title to be more descriptive and incorporated your other comments.

**Comment by aiwilliams** [comment]
> I have not noticed the enable all sub-accounts issue, does this happen when it is set to disabled on initial config?

Yup, confirmed in https://bitbucket.org/jupiterone/jupiter-integration-aws/src/a0f84ef2d4339b9b196f1aab09e29c2291961220/src/integration-aws/synchronizer/account.ts#lines-270.

I really don't think that what is here is incorrect, so I think we should merge this PR.

---

To be complete, we would need to also communicate something like:

* Configure the master account integration instance WITHOUT checking the box
* Enable the integration at the desired polling interval
* Provision the sub accounts, tag as desired
* When happy with AWS account provisioning...
* Modify integration config by checking the box to auto-create sub account integration configs

Also, we may want to support the case where they don't have the master instance enabled, do this provisioning, manually run the master and have all other accounts set up and remain disabled until they feel comfortable enabling one at a time. We may also need an "enable all" button as they tire of walking down the list and have confidence. 😄

**Review by aiwilliams — APPROVED** [review]

File: faqs/faqs-aws.md [hunk 2]
```diff
@@ -68,3 +86,27 @@ You can then build queries using these tag properties. For example:
...
+allow access to all resources in the s3 service and those that allow access to all services in the aws_account.
+
+This is done simply as follows:
+
+```
+Find AccessPolicy 
+  that ALLOWS (aws_account|aws_s3|aws_s3_bucket) ...
+```
+
+[jupiterone-aws-integration]: https://github.com/jupiterone/jupiterone-aws-integration
```
Comment (aiwilliams) [inline faqs/faqs-aws.md:112]: 👍

**Review by aiwilliams — APPROVED** [review]

**Comment by a-u-h-g** [comment]
Nice, I added some changes per your process so no one yells at us for the polling interval ;)

**Pull request merged**